### DISCUSSION TITLE
left fixed scrolling menu cant have fixed top block

### DISCUSSION THREAD

**Discussion by zdarin** [fomantic/Fomantic-UI#1360] [Collections (form, grid, menu, message, table, ...)] [answered]
Hello, i cant figure out how to make left fixed menu with top fixed block on top of left menu block. Because  when too much buttons on left fixed menu i want scroll only menu buttons. 
Or i something missed?? Can someone help me figure out this?

<a href="https://ibb.co/njmRrgg"><img src="https://i.ibb.co/XtLVX22/Screenshot-2020-03-10-at-12-57-44.png" alt="Screenshot-2020-03-10-at-12-57-44" border="0"></a>

**Comment by lubber-de** [comment]
Could you please share the code in a jsfiddle?

**Comment by zdarin** [comment, marked as answer]
i only figured out like this working https://jsfiddle.net/w9dfcv07/1/

**Comment by lubber-de** [comment]
This looks pretty much like the solution you were searching for

**Comment by zdarin** [comment]
but this correct solution or its must be in one block??

**Comment by lubber-de** [comment]
As the menu component does not provide this out of the box, i think this solution is ok.

**Comment by zdarin** [comment]
ok thank you
